mypy/server/deps.py

```diff
@@ -172,11 +189,13 @@ def visit_func_def(self, o: FuncDef) -> None:
             else:
                 signature = o.type
             for trigger in get_type_triggers(signature):
+                # TODO: avoid adding both `mod.func` and `<mod.func>` here and below for aliases
                 self.add_dependency(trigger)
                 self.add_dependency(trigger, target=make_trigger(target))
         if o.info:
             for base in non_trivial_bases(o.info):
                 self.add_dependency(make_trigger(base.fullname() + '.' + o.name()))
+        self.add_type_alias_deps(self.scope.current_full_target(), add_trigger=True)
```

Comment:
Add comment about why we have `add_trigger=True`. Again, I believe that this should be `current_target()` instead of `current_full_target()`.